datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedPartitionAssigner.java

```diff
@@ -29,7 +40,15 @@
 /**
  * Performs partition assignment based on partition throughput information
  */
-public class LoadBasedPartitionAssigner {
+public class LoadBasedPartitionAssigner implements MetricsAware {
+  private static final Logger LOG = LoggerFactory.getLogger(LoadBasedPartitionAssignmentStrategy.class.getName());
```

Comment:
The class name for logging and metrics is set to `LoadBasedPartitionAssignmentStrategy` on purpose. It will be easier to filter for these along with logs and metrics from the main strategy class if they share the same namespace.